dspace-api/src/main/resources/org/dspace/storage/rdbms/sqlmigration/postgres/V7.0_2019.11.13__relationship_type_copy_left_right.sql

```diff
@@ -0,0 +1,14 @@
+--
+-- The contents of this file are subject to the license and copyright
+-- detailed in the LICENSE and NOTICE files at the root of the source
+-- tree and available online at
+--
+-- http://www.dspace.org/license/
+--
+
+-----------------------------------------------------------------------------------
+-- Create columns copy_left and copy_right for RelationshipType
+-----------------------------------------------------------------------------------
+
+ALTER TABLE relationship_type ADD copy_left BOOLEAN DEFAULT FALSE NOT NULL;
+ALTER TABLE relationship_type ADD copy_right BOOLEAN DEFAULT FALSE NOT NULL;
```

Comment:
I agree with @paulo-graca that the names CopyLeft and CopyRight are vague.  I just wanted to point out that we should also correct them at the database level.  These should be renamed `copy_to_left` and `copy_to_right`  (assuming I have the meaning correct).

Reply:
This has been updated